spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe 'Show', type: :feature do
+  
+  let!(:echo) { Hotel.create!(name: 'Echo Mountain Inn', location: 'Echo Mountain') }
+  let!(:winterpark) { Hotel.create!(name: 'Winter Park Hotel', location: 'Winter Park') }
+
+  let!(:andra) { Guest.create!(name: "Andra", nights: 4) }
```

Comment:
Nice job with this data setup for your tests. I would try to make at least 1 more Guest that you can use to play with in your scenarios; one is ok but more data is always good 👍